mreg/api/v1/tests/tests.py

```diff
@@ -849,7 +883,7 @@ def test_ptr_override_ipv6_patch_204(self):
 
     @skip("This test crashes the database and is skipped "
           "until the underlying problem has been fixed")
-    def test_ptr_override_reject_invalid_ipv4_400(self):
+    def test_ptr_override_reject_invalid_ipv4_400(self):  # pragma: no cover
```

Comment:
Why? Add comment or fix testing.

Reply:
The test is currently skipped. Coverage complains that we don't test the test, since we don't run it.

Reply:
Commented on skipped tests in https://github.com/unioslo/mreg/pull/484/commits/3fb4213c07bbd979d25a4cf18a83dae61abdb22b